IssueTracker/IssueTracker.FileSystem/Data/IData/IMetaData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Models.Request;
+using Models.Response;
+
+namespace IssueTracker.FileSystem.Data.IData;
+
+public interface IMetaData
+{
+    IEnumerable<MetaDataResponse> GetAll();
+    Task CreateOrUpdateAsync(MetaDataRequest entity);
+    Task<bool> DeleteAsync(string id, string group);
+    Task<MetaDataResponse?> GetAsync(string rowKey, string partitionKey);
```

Comment:
RowKey and PartitionKey are implementation details and should not be exposed here

Reply:
I changed